docs/operator-manual/reconcile.md

```diff
@@ -111,3 +111,9 @@ data:
     # actually changing in content.
     - .status.conditions[].lastTransitionTime
 ```
+
+### Analyzing logs to find slow steps of reconciliation
```

Comment:
I am not sure the comment provide enough information to significantly help debugging. The datadog example depends on the underlying infrastructure of the Argo users and how that infrastructure index logs. Slack is a good place to share splunk/datadog query examples.

What would be interesting at the performance level would be to document all the relevant `_ms` that could help users detect performance problems. For each of them, provide an explanation of what could be the known possible causes of slowness. This doc could most likely lives in a `docs/operator-manual/performance_investigation.md` on it's own.

Reply:
+1 on the Arithmetic interpreted note. That would def be useful